main.go

```diff
@@ -175,23 +181,22 @@ func main() {
 
 	leClient, err := kubernetes.NewForConfig(rest.AddUserAgent(cfg, "leader-election"))
 	if err != nil {
-		klog.Fatal(err)
+		klog.Fatalf("Error creating leader election kubernetes clientset: %s", err.Error())
 	}
 
 	eventBroadcaster := record.NewBroadcaster()
-	eventBroadcaster.StartLogging(klog.V(4).Infof)
+	eventBroadcaster.StartLogging(klog.V(log.DEBUG).Infof)
```

Comment:
@tpantelis We are changing this from INFO (i.e., log level 4) to DEBUG (i.e., 2). I hope this is not a typo?

Reply:
We're collapsing all previous levels (eg 2, 4, 6, 8) to just 2. I thought DEBUG was appropriate here - do you think it should be TRACE (3)?

Reply:
I think its fine.